app/schemas.py

```diff
@@ -304,6 +305,15 @@ def validate_permissions(self, value):
             duplicates = list(set([x for x in permissions if permissions.count(x) > 1]))
             raise ValidationError('Duplicate Service Permission: {}'.format(duplicates))
 
+    @validates('email_from')
+    def validate_email_from(self, value):
+        if not all(
+            char in string.ascii_lowercase + string.digits + '.' for char in value
+        ):
```

Comment:
A totally optional suggestion if you think it’s more elegant:
```suggestion
        if set(value).difference(
            string.ascii_lowercase + string.digits + '.'
        ):
```

Reply:
It's more elegant, though I _think_ what I wrote is maybe a touch more readable. I will leave it since you said it's optional :)